sqlglot/dialects/mysql.py

```diff
@@ -823,6 +823,294 @@ class Generator(generator.Generator):
             exp.DataType.Type.TIMESTAMPLTZ,
         }
 
+        RESERVED_KEYWORDS = {
+            "accessible",
+            "add",
+            "all",
```

Comment:
Our MySQL-like dialect uses "all" as a special table.

😄 🔫 

I think we can just pop this out in our subclass though, so no worries

Reply:
Haha damn... Well, it'd still work I think, but with this PR you'd get quotes in the name because you can't do:

```sql
SELECT 1 AS all
```